src/uu/install/src/install.rs

```diff
@@ -483,40 +485,73 @@ fn is_new_file_path(path: &Path) -> bool {
             || path.parent().unwrap().as_os_str().is_empty()) // In case of a simple file
 }
 
+/// Test if the path is an existing directory or ends with a trailing separator.
+///
+/// Returns true, if one of the conditions above is met; else false.
+///
+#[cfg(unix)]
+fn is_potential_directory_path(path: &Path) -> bool {
+    let separator = MAIN_SEPARATOR as u8;
+    path.as_os_str().as_bytes().last() == Some(&separator) || path.is_dir()
+}
+
+#[cfg(not(unix))]
+fn is_potential_directory_path(path: &Path) -> bool {
+    let path_str = path.to_string_lossy();
+    path_str.ends_with(MAIN_SEPARATOR) || path_str.ends_with('/') || path.is_dir()
```

Comment:
I'm wondering if the second case here is really expected behaviour. Does windows accept trailing `/` in paths?

Reply:
ping ?

Reply:
> Just one question, otherwise ready to be merged I think! Sorry for taking so long :)

No worries! We all have personal lives sometimes :)



> I'm wondering if the second case here is really expected behaviour. Does windows accept trailing `/` in paths?

AFAIK it does. I'm not sure, whether that is a valid delimiter in the standard cmd.exe or powershell already, or if this is only allowed when you use a MinGW or Git-bash shell. Nonetheless you can find the same check already in here `uucore::features::fs::canonicalize()`.

Reply:
```
    let has_to_be_directory =
        (miss_mode == MissingHandling::Normal || miss_mode == MissingHandling::Existing) && {
            let path_str = original.to_string_lossy();
            path_str.ends_with(MAIN_SEPARATOR) || path_str.ends_with('/')
        };
```

Reply:
Alright! Sounds good. We can merge this if you fix the conflicts.

Reply:
The branch is now up to date. Still merging is not possible. 
Am I missing something @sylvestre and @tertsdiepraam ?

Reply:
Maybe the main of your repo is not up to date with the upstream one? I usually do something like this to ensure my main is set to the upstream one:
```
git switch main
git fetch upstream
git reset --hard upstream/main
```